plugin/route53/setup.go

```diff
@@ -31,6 +31,22 @@ var f = func(credential *credentials.Credentials) route53iface.Route53API {
 	return route53.New(session.Must(session.NewSession(&aws.Config{Credentials: credential})))
 }
 
+func parseDurationFromCaddyController(cc *caddy.Controller, def time.Duration) (duration time.Duration, err error) {
```

Comment:
this doesn't need a extra function just to parse a duration

Reply:
will remove, this was for when I added another option for refreshing. but since it's not needed anymore, I will revert the file to the original contents.